geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-parquet/src/test/resources/geoparquet-metadata-schema.json

```diff
@@ -0,0 +1,125 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "GeoParquet",
+  "description": "Parquet metadata included in the geo field.",
+  "type": "object",
+  "required": ["version", "primary_column", "columns"],
+  "properties": {
+    "version": {
+      "type": "string",
+      "const": "1.1.0-dev"
```

Comment:
we should be using https://geoparquet.org/releases/v1.0.0/schema.json here, not the dev one